Tests/test_file_dcx.py

```diff
@@ -11,19 +14,35 @@ def test_sanity(self):
         # Arrange
 
         # Act
-        im = Image.open(TEST_FILE)
+        with Image.open(TEST_FILE) as im:
 
-        # Assert
-        self.assertEqual(im.size, (128, 128))
-        self.assertIsInstance(im, DcxImagePlugin.DcxImageFile)
-        orig = hopper()
-        self.assert_image_equal(im, orig)
+            # Assert
+            self.assertEqual(im.size, (128, 128))
+            self.assertIsInstance(im, DcxImagePlugin.DcxImageFile)
+            orig = hopper()
+            self.assert_image_equal(im, orig)
 
+    @unittest.skipIf(sys.version_info.major < 3 or is_pypy, "Requires CPython 3")
```

Comment:
As Python 2 support is being dropped, should I go ahead and remove this version check now?

Reply:
It's already done in #4109.

Reply:
Yeah, go for it.

Reply:
Done.